Portfolio2.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+   <head>
+      <meta charset="UTF-8">
+      <title>Elmi Farah's Portfolio'</title>
+      <meta name="viewport" content="width=device-width, initial-scale=1">
+      <link rel="stylesheet" type="text/css " href="portfolio2.css ">
+   </head>
+   <body>
+      <header>
+         <div id="bannertop">
+            <div class="menue">
```

Comment:
Do you need these two divs? Id suggest removing them to simplify your markup. Make sure you update the css accordingly as just deleting them will break your menu.